src/js/providers/providers-register.ts

```diff
@@ -1,9 +1,10 @@
 import ProvidersLoaded from 'providers/providers-loaded';
 import ProvidersSupported from 'providers/providers-supported';
 import DefaultProvider from 'providers/default';
+import type VideoProvider from 'providers/html5';
```

Comment:
we should replace this with https://github.com/jwplayer/jwplayer/pull/3636/files#diff-cae9122c8d4f001c56093abab4fae68eR75 when this merged